code/core/src/manager/container/Menu.tsx

```diff
@@ -140,49 +141,18 @@ export const useMenu = (
       onClick: () => api.togglePanel(),
       active: isPanelShown,
       right: enableShortcuts ? <Shortcut keys={shortcutKeys.togglePanel} /> : null,
-      icon: isPanelShown ? <CheckIcon /> : null,
+      icon: isPanelShown ? <CheckIcon /> : <></>,
     }),
     [api, enableShortcuts, shortcutKeys, isPanelShown]
   );
 
-  const addonsOrientationToggle = useMemo(
-    () => ({
-      id: 'D',
-      title: 'Change addons orientation',
-      onClick: () => api.togglePanelPosition(),
-      right: enableShortcuts ? <Shortcut keys={shortcutKeys.panelPosition} /> : null,
-    }),
-    [api, enableShortcuts, shortcutKeys]
-  );
-
-  const fullscreenToggle = useMemo(
-    () => ({
-      id: 'F',
-      title: 'Go full screen',
-      onClick: () => api.toggleFullscreen(),
-      active: isFullscreen,
-      right: enableShortcuts ? <Shortcut keys={shortcutKeys.fullScreen} /> : null,
-      icon: isFullscreen ? <CheckIcon /> : null,
-    }),
-    [api, enableShortcuts, shortcutKeys, isFullscreen]
-  );
-
-  const searchToggle = useMemo(
-    () => ({
-      id: '/',
-      title: 'Search',
-      onClick: () => api.focusOnUIElement(focusableUIElements.storySearchField),
-      right: enableShortcuts ? <Shortcut keys={shortcutKeys.search} /> : null,
-    }),
-    [api, enableShortcuts, shortcutKeys]
-  );
```

Comment:
Are these deleted/re-ordered for a reason?

Reply:
Yes, according to updated designs by @MichaelArestad